README.md

```diff
@@ -5,36 +5,25 @@
 As an example of a Release CR:
 
 ```
-apiVersion: "release.giantswarm.io/v1alpha1"
-kind: "Release"
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
 metadata:
-  name: "azure.v0.0.1"
-  labels:
-    giantswarm.io/managed-by: "app-operator"
-    giantswarm.io/provider: "azure"
+  creationTimestamp: null
+  name: v11.3.0
 spec:
-  changelog:
-    - component: "azure-operator"
-      description: "Updated to foo the bar."
-      kind: "changed"
+  apps:
+  - name: app-1
+    version: 1.2.1
   components:
-    - name: "azure-operator"
-      version: "0.0.1"
-  parentVersion: "0.0.0"
-  version: "0.0.1"
+  - name: ignore-me
```

Comment:
Could add `// releaseOperatorDeploy: false` just to make it clear why it's being ignored.

Reply:
Updated.